client/wfprev-war/src/main/angular/src/app/services/spatial-services.ts

```diff
@@ -242,23 +246,28 @@ export class SpatialService {
     }
 
     extractGDBGeometry(file: File): Observable<any> {
-        // this will be replaced with node js server in AWS - WFPREV-402
-        const url = 'http://localhost:3000/upload';
-
+        // this will not work locally as it is connecting to AWS SDK. 
+        // testing locally with higher environments will also give a CORS error.
+        const url = `${this.appConfigService.getConfig().rest['wfprev']}/wfprev-api/gdb/extract`;
         const formData = new FormData();
         formData.append('file', file);
+        const headers = new HttpHeaders({
+            Authorization: `Bearer ${this.tokenService.getOauthToken()}`
```

Comment:
Not a issue but just curious why do we need token to acess the gdp-extractor?

Reply:
Because it's being served through the wfprev-api, to protect against DoS attacks, excessive hits on the lambda etc. We don't want any human accessing the lambda directly. Just the java API should be accessible